app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java

```diff
@@ -60,21 +66,44 @@ public void afterTextChanged(Editable s) {
         vLoginButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                Toast.makeText(getContext(), "Log In press", Toast.LENGTH_LONG).show();
-                //getPresenter().storeUsername(vUserNameInput.getText().toString());
+                if (TextUtils.isEmpty(vUserNameInput.getText())) {
```

Comment:
Maybe we can move this to a new method. like IsValidUsernameAndPassword() that return a boolean to do something

Reply:
Done!